Software/Python/grovepi_i2c.py

```diff
@@ -1,66 +0,0 @@
-#!/usr/bin/env python
```

Comment:
i'm hesitant to remove it. Is there a conflict ?

Reply:
No conflict. It does nothing. It's not called anywhere.